weather_finder/weather_finder.py

```diff
@@ -3,47 +3,44 @@
 API_KEY = '9104e0a25f676b56ac31901d10c5cf0a'
 
 
-def verify_city(json_response):
-    """
-    Takes in response from OWM and checks if city exists
-
-    Arguments:
-       json_response (json): contains response of a request from OWM
-    """
-
-    if json_response['cod'] == '404':
-        print 'City could not be found, please retry'
-	return False
-    else:
-	print 'City found'
-	return True
-
-
 def get_weather_info(location):
     """
-    Gets information about the weather of a location and displays it
+    Gets information about the weather of a location and displays it.
 
     Arguments:
         location (str): a string that specifies which place to search weather
     """
-	
-    location_url = 'http://api.openweathermap.org/data/2.5/weather?q=' + location + '&APPID=' + API_KEY	
+    location_url = 'http://api.openweathermap.org/data/2.5/weather?q=' + location + '&APPID=' + API_KEY
     weather_raw_data = requests.get(location_url)
     weather_json = weather_raw_data.json()
     if not verify_city(weather_json):
-	return
+        return
     else:
-	data_output(data_organizer(weather_json))
- 
+        weather_data_output_on_console(weather_data_organizer(weather_json))
```

Comment:
We should return the parsed weather data instead of outputting to console directly, since the method name is `get_weather_info` as the reader would assume that this method should return something.